WordPress/Classes/Utility/Blogging Reminders/BloggingReminders.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+/// Main interface for scheduling blogging reminders
+///
+class BloggingReminders {
+
```

Comment:
Maybe we can even do a protocol and then a concrete type for the weekday schedule? Or something like that?

Reply:
I didn't understand this suggestion.  Can I ask you to expand, or to ping me via DM to chat about it a bit more?

Reply:
Sorry the comment was totally unclear. I had in mind that this is a scheduler, so I wondered if we could do a `Scheduler` protocol for more general purposes and then a concrete implementation of `BloggingReminderScheduler`. On the other hand, I then noticed that we already have a concrete type called `Scheduler` so, reconsidering it all, it might be just me overthinking. In short, I think this comment can be ignored for now 😀

Reply:
Thanks, that makes it clearer.  If you're ok with it I'll move ahead as it is for now, and then we can iterate on these classes as needed.

Reply:
yes 100%. Thank you!!

Reply:
Marking as resolved for now.